src/localization/fi.json

```diff
@@ -137,9 +135,23 @@
   "editableFields": {
     "htmlField": "Html-sisältö",
     "submitFieldText": "Lähetä-napin teksti",
+    "radioFieldText": "Radiokentän teksti",
     "numberField": "Numerokenttä",
     "numberFieldMin": "Aseta minimiarvo",
-    "numberFieldMax": "Aseta maksimiarvo"
+    "numberFieldMax": "Aseta maksimiarvo",
+    "default": {
+      "label": "label",
+      "html": "html",
+      "text": "text",
+      "radio": "radio",
+      "select": "select",
+      "number": "number",
+      "memo": "memo",
+      "checkList": "checkList",
+      "submit": "submit",
+      "dateTime": "dateTime",
+      "autoComplete": "autoComplete"
```

Comment:
Once the visual display components are ready, strings should be localized correctly